pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
```

Comment:
Need to add a `build-system` section here.

For Poetry this is:
```
[build-system]
requires = ["poetry>=1.0.0"]
build-backend = "poetry.mason.api"
```

For Pip it would be:

```
[build-system]
requires = ["setuptools", "wheel"]
build-backend = "setuptools.build_meta"
```